data/hai/hai reveal 6 end.txt

```diff
@@ -317,25 +335,45 @@ mission "Hai Reveal [C05] Cleanup"
 			`	She gives you a hard stare, but her resolve isn't up to the task for more than a couple seconds. She looks away, and you can see her blinking away more tears. "Is this to be my end, then?" she asks. "Is this the price I pay for my efforts?"`
 			label decide
 			choice
-				`	(Spare her.)`
-					goto spare
-				`	(End her.)`
-					goto end
 				`	"Am I to be judge, jury, and executioner?"`
 					goto judge
+					to display
+						not "Hai Reveal [C05] Cleanup: label judge"
+				`	(Spare her.) "The whole galaxy hunts you now. Let that be your justice."`
```

Comment:
Since the whole point is that she's not actually hunted, you're the only person who knows so long as she stays hidden, it's not really appropriate to say she's hunted.
```suggestion
				`	(Spare her.) "There is no other home for you now. Let that be your justice."`
```